lib/routes/account.js

```diff
@@ -194,14 +197,20 @@ module.exports = function (
         }
 
         function createSessionToken () {
+          doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(account, config, request)
+
+          // Create a verified session if preVerified flag set or sign-in confirmation disabled
+          var tokenVerifiedId = (preVerified || !doSigninConfirmation) ? undefined : tokenVerificationId
```

Comment:
Using a fresh `var` with a slightly-different name feels a bit funny to me here. Would it be clearer instead if the condition was reworked to simply unset `tokenVerificationId`? Something like:

```js
if (preVerified || ! doSigninConfirmation) {
  tokenVerificationId = undefined
}
```